makefiles/utils/ansi_special.mk

```diff
@@ -0,0 +1,16 @@
+# Make does not recognize escaped characters (e.g. \t, \033, etc).
+# The following variables contain the characters themselves.
+# This file is inherently fragile (i.e. your editor could destroy the tab
+# without you realizing, to be careful when editing this.
+
+# To generate this: $ printf "ANSI_ESC := \033# comment"
+ANSI_ESC := # you may or may not be able to see that character in your editor
+
+# The third parameter to the subst is a tab - be careful not to replace it with
+# spaces!
+TAB := $(subst ,,	)
+
+define NEWLINE
+
+
+endef
```

Comment:
These are not being used so far, right?

Reply:
Nope, they were used in https://github.com/RIOT-OS/RIOT/pull/12156/files when replacing all occurrences of `COLOR_ECHO` and others, but I did not want to go into the trouble of that rebase ATM.

Reply:
So, we are keeping them because that would be the next step?

Reply:
Yep, to remove the `COLOR_ECHO` usage this would be handy, but I can remove if you prefer.